athena-federation-sdk/src/main/java/com/amazonaws/athena/connector/lambda/serde/v2/ListTablesRequestSerDe.java

```diff
@@ -78,7 +85,16 @@ protected MetadataRequest doRequestDeserialize(JsonParser jparser, Deserializati
         {
             String schemaName = getNextStringField(jparser, SCHEMA_NAME_FIELD);
 
-            return new ListTablesRequest(identity, queryId, catalogName, schemaName);
+            String nextToken = null;
+            int pageSize = -1;
```

Comment:
may want to add a comment for why you are doing this. Also, we may want to be stricter about this based on serialization version.

Reply:
Added a `TODO` comment to enforce parsing when we upgraded to V3.